docs/ansible.utils.to_paths_filter.rst

```diff
@@ -52,7 +52,7 @@ Parameters
                     <td>
                     </td>
                 <td>
-                        <div>Prepend each path entry. Useful to add the initial <code>var</code> name.</div>
+                        <div>Prepend each path entry. Useful to add the initial <code>var</code>. name.</div>
```

Comment:
This looks odd - should there be another sentence at the end? One that ends with `name`?